### ISSUE TITLE
Filebeat README has bad links

### ISSUE THREAD

**Issue by markwalkom** [elastic/beats#5710]
Reported here - https://twitter.com/z3ndrag0n/status/934354610650615808

If you grab https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-6.0.0-darwin-x86_64.tar.gz and extract it, the `README.md` has 4 links that all 404.

File - https://www.elastic.co/guide/en/beats/filebeat/6/filebeat-getting-started.html
Actual - https://www.elastic.co/guide/en/beats/filebeat/6.0/filebeat-getting-started.html

Basically, all 3 links need a `.0` added to specifically point them to 6.0 docs.

I wasn't sure how this was generated, ie if it was automatic, so didn't want to make a PR against the file.

**Comment by ruflin** [comment]
The file is generated out of https://github.com/elastic/beats/blob/master/dev-tools/packer/readme.md.j2 and it should contain the value from here: https://github.com/elastic/beats/blob/6.0/libbeat/docs/version.asciidoc#L2 (which is 6.0). I need to investigate was is going wrong here during packaging.

**Comment by ruflin** [comment]
I opened https://github.com/elastic/beats/pull/5715 against 6.0. The issue is that gotpl convert 6.0 to 6, that is why we have the issue only with all first major version and we never had it in the past, as we didn't have that script yet for 5.0.

**Comment by dedemorton** [comment]
Closing this issue because it was fixed in #5715 plus I've had redirects set up.
